scripts/istio-images-mirror

```diff
@@ -14,3 +14,8 @@ pstauffer/curl rancher/pstauffer-curl v1.0.3
 coredns/coredns rancher/coredns-coredns 1.1.2
 quay.io/coreos/prometheus-operator rancher/coreos-prometheus-operator v0.32.0
 quay.io/coreos/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.32.0
+plugins/docker rancher/plugins-docker 18.09
```

Comment:
This image has the same source as jenkins-plugins-docker, which is already in Rancher dockerhub https://github.com/rancher/types/blob/master/image/mirror.go#L17. Is the intent to re-name the image?

Reply:
Yes, it is mistakenly mirrored to `jenkins-plugins-docker` but it is not a jenkins plugin(drone plugin image indeed), so I think we should correct the name